kpi/tests/api/v1/test_api_submissions.py

```diff
@@ -79,17 +90,22 @@ def test_list_submissions_as_owner_with_params(self):
                 'limit': 5,
                 'sort': '{"q1": -1}',
                 'fields': '["q1", "_submitted_by"]',
-                'query': '{"_submitted_by": {"$in": ["", "someuser", "another"]}}',
+                'query': '{"_submitted_by": {"$in": ["unknownuser", "someuser", "anotheruser"]}}',
```

Comment:
The test is just to ensure that query works with param even as the owner. 
`self.asset` does not allow anonymous user to `POST` data. Since, `mock_submissions` tries to simulate real data submission, we would need to add another test where `self.asset.deployment.xform.require_auth = False` .